src/dotnet/APIView/APIViewWeb/Languages/PythonLanguageService.cs

```diff
@@ -17,29 +17,27 @@ public class PythonLanguageService : LanguageProcessor
     {
         public override string Name { get; } = "Python";
         public override string Extension { get; } = ".whl";
-        public override string VersionString { get; } = "0.2.8";
+        public override string VersionString { get; } = "0.2.11";
```

Comment:
What is this needed for (I can see I haven't updated it--oops)?

Reply:
we already published 0.2.10 to feed. I need latest changes in this PR to integrate published package to APIVIew to run stubgen as a module. So only option is to publish a new version and refer the same when generating/ checking if update is required.

Reply:
This property is still used to check if we need to auto update an existing review or not.